src/Umbraco.Web.Common/Mvc/UmbracoMvcConfigureOptions.cs

```diff
@@ -17,11 +17,8 @@ namespace Umbraco.Cms.Web.Common.Mvc;
 public class UmbracoMvcConfigureOptions : IConfigureOptions<MvcOptions>
 {
     public UmbracoMvcConfigureOptions()
-    { }
-
-    [Obsolete("The global settings is not required anymore, use the default constructor instead. Scheduled for removal in Umbraco 17.")]
-    public UmbracoMvcConfigureOptions(IOptions<GlobalSettings> globalSettings)
-    { }
+    {
```

Comment:
I think you can simply remove this constructor now, given it's not accepting any arguments.